packages/non-core/coffeescript/package.js

```diff
@@ -6,15 +6,15 @@ Package.describe({
   // so bumping the version of this package will be how they get newer versions
   // of `coffeescript-compiler`. If you change this, make sure to also update
   // ../coffeescript-compiler/package.js to match.
-  version: '1.12.7_2'
+  version: '1.12.7-2-beta.30'
```

Comment:
Congratulations to @GeoffreyBooth on [the release of CoffeeScript 2](https://news.ycombinator.com/item?id=15276833)! Thanks for all your efforts to ensure CoffeeScript has first-class support in Meteor.